### ISSUE TITLE
EpubCheck doesn't report missing properties="svg" anymore since 4.0

### ISSUE THREAD

**Issue by tofi86** [IDPF/epubcheck#716]
EpubCheck 3.0.1 (which Apple still uses internally) reports:

```
Epubcheck Version 3.0.1

Validating against EPUB version 3.0
ERROR:ebook.epub/OEBPS/chapter9.xhtml: This file should declare in the OPF the property: svg

Check finished with warnings or errors
```

Where EpubCheck 4.0 and 4.0.1 report a valid EPUB 3:
```
Validating using EPUB version 3.0.1 rules.
No errors or warnings detected.
epubcheck completed
```

I would expect EpubCheck 4.0.1 to report the missing svg properties entry as well.

**Comment by mattgarrish** [comment]
Whether it is an error depends on how the SVG is included. The property definition says:

> This property must be set when SVG markup is included directly in the resource and may be set when the SVG is referenced from the resource (e.g., from an [HTML] img, object or iframe element).

**Comment by tofi86** [comment]
Hi Matt, thanks for clarifying this issue. In my case the SVG was an image reference, so we should check first whether the changes from 3.0.1 to 4.0.0 were applied correctly...
Adding the review label...
